src/main/webapp/queue/student.html

```diff
@@ -90,6 +128,30 @@
       });
     }
 
+    function sendNotification(){
```

Comment:
One possible extension to this could be to send the notification from the server if possible. That could get rid of the need for pollong.